core/src/main/java/tachyon/worker/eviction/EvictStrategies.java

```diff
@@ -28,8 +28,12 @@
    */
   public static EvictStrategy getEvictStrategy(EvictStrategyType strategyType, boolean isLastTier) {
     switch (strategyType) {
+      case LFU:
```

Comment:
Maybe we should move to making this more pluggable.  Could you file a jira to use either configs or register like ufs PR? On phone so can't access jira